composables/useGraphql.ts

```diff
@@ -28,17 +28,19 @@ export default function ({
   queryName,
   clientName = '',
   variables = {},
-  options = {},
   disabled = computed(() => false),
   data = ref(),
   error = ref(),
   loading = ref(true),
 }) {
+  const { client: clientPrefix } = usePrefix()
   const { $consola } = useNuxtApp()
-  const { prefix, client } = useQueryParams({ queryPrefix, clientName })
```

Comment:
we don't need `useQueryParams` anymore?

Reply:
yes, I move it under `export default function`